Dockerfile

```diff
@@ -0,0 +1,19 @@
+FROM ubuntu:latest
+LABEL authors="lucasliebe"
+
+WORKDIR /app
+
+RUN apt-get update -y && apt-get install default-jre make git wget curl unzip build-essential -y
```

Comment:
If we need this many dependencies for the setup I think it would be better to make a multilayer image, i.e. one layer for building and loading everything and another one to just run the web interface. This way the final image will be much smaller.

Reply:
Working on it